assets/js/components/KeyMetrics/ConnectModuleCTATile.js

```diff
@@ -72,7 +72,7 @@ export default function ConnectModuleCTATile( { moduleSlug } ) {
 							module.name
 						) }
 					</p>
-					<Link href="#" onClick={ handleConnectModule }>
+					<Link secondary onClick={ handleConnectModule }>
```

Comment:
I've added the `secondary` prop in order to maintain the correct colour for the link.